docs/ide/tutorial-csharp-aspnet-core.md

```diff
@@ -19,7 +19,7 @@ ms.workload:
 ---
 # Tutorial: Get started with C# and ASP.NET Core in Visual Studio
 
-In this tutorial for C# development with ASP.NET Core using Visual Studio, you'll create a C# ASP.NET Core web app, make changes to it, explore some features of the IDE, and then run the app.
+In this tutorial for C# development with ASP.NET Core using Visual Studio, you will learn how you can create, make changes to and run a C# ASP.NET Core web app. You will also get to explore some features of the Visual Studio IDE.
 
```

Comment:
@jmayntzhusen I must respectfully decline this change. We've got internal guidelines that we follow for voice and one of those is to use contractions. Please see [Project friendliness](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#project-friendliness) section of the [Top 10 tips for Microsoft style and voice](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice) topic in the Microsoft Style Guide for more information. Otherwise, while we appreciate another outlook on the words we've selected, we do try to not add extra words if they aren't necessary, as the first sentence is what typically shows up in search results and those are limited by a character count.